File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/body/commun/VerificationCodeBody.java
package com.lattis.ellipse.data.network.model.body.commun;

import com.google.gson.annotations.SerializedName;

public class VerificationCodeBody {

    @SerializedName("user_id")
    private String user_id;

    @SerializedName("account_type")
    private String account_type;

    @SerializedName("confirmation_code")
    private String confirmationCode;

    public VerificationCodeBody(String user_id, String account_type, String confirmationCode) {
        this.user_id = user_id;
        this.account_type = account_type;
        this.confirmationCode = confirmationCode;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/authentication/VerificationCodeBody.kt
package com.lattis.data.entity.body.authentication

import com.google.gson.annotations.SerializedName

class VerificationCodeBody(
    @field:SerializedName("user_id") private val user_id: String?,
    @field:SerializedName("account_type") private val account_type: String?,
    @field:SerializedName("confirmation_code") private val confirmationCode: String?
)